doc/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Change Log
 
+## 1.16.1
+
+Released on 27th May 2020
+
+Updated to @polkadot/api package v1.16.1
```

Comment:
mention the `derive` package as well

Reply:
Changed in d451e570404b68cec482591b50a084b7d41f4701